src/FSharp.Core/list.fsi

```diff
@@ -1449,6 +1625,9 @@ module List =
     [<CompiledName("MapIndexed2")>]
     val mapi2: mapping:(int -> 'T1 -> 'T2 -> 'U) -> list1:'T1 list -> list2:'T2 list -> 'U list
 
+    /// 
+    /// Time complexity: O(n) - where n is the list length
+    /// Space complexity: O(1)
```

Comment:
The added complexity information should be within the `<summary>` tag, right before the closing tag.